### PR TITLE
Release/4.3.4

### LINKED ISSUE #175: Release version 4.3.4
This issue is for tracking changes for the 4.3.4 release.  Target release date: **10 January 2023**.

### [Release steps](https://github.com/10up/windows-azure-storage/blob/develop/CONTRIBUTING.md#release-instructions)

- [x] Branch: Starting from `develop`, cut a release branch named `release/4.3.4` for your changes.
- [x] Version bump: Bump the version number in `js/windows-azure-storage-admin.js`, `js/windows-azure-storage-media-browser.js`, `package.json`, `package-lock.json`, `readme.txt`, and `windows-azure-storage.php` if it does not already reflect the version being released.  Update both the plugin "Version:" property and the plugin `MSFT_AZURE_PLUGIN_VERSION` constant in `windows-azure-storage.php`.
- [x] Changelog: Add/update the changelog in both `readme.txt` and `CHANGELOG.md`.
- [x] Props: Update `CREDITS.md` file with any new contributors, confirm maintainers are accurate.
- [x] New files: Check to be sure any new files/paths that are unnecessary in the production version are included in `.distignore`.
- [x] Readme updates: Make any other readme changes as necessary.  `README.md` is geared toward GitHub and `readme.txt` contains WordPress.org-specific content.  The two are slightly different.
- [x] Merge: Make a non-fast-forward merge from your release branch to `develop` (or merge the pull request), then do the same for `develop` into `trunk` (`git checkout trunk && git merge --no-ff develop`).  `trunk` contains the stable development version.
- [x] ADD NEW ZIP ACTION AND UPDATE RELEASE INSTRUCTIONS
- [x] Push: Push your trunk branch to GitHub, e.g. `git push origin trunk`.
- [x] Release: Create a [new release](https://github.com/10up/windows-azure-storage/releases/new), naming the tag and the release with the new version number, and targeting the `trunk` branch. Paste the changelog from `CHANGELOG.md` into the body of the release and include a link to the closed issues on the milestone (e.g. `https://github.com/10up/windows-azure-storage/milestone/9?closed=1`).
- [x] SVN: Wait for the [GitHub Action](https://github.com/10up/windows-azure-storage/actions) to finish deploying to the WordPress.org repository.  If all goes well, users with SVN commit access for that plugin will receive an emailed diff of changes.
- [x] Check WordPress.org: Ensure that the changes are live on [https://wordpress.org/plugins/windows-azure-storage/](https://wordpress.org/plugins/windows-azure-storage/). This may take a few minutes.
- [x] Close milestone: Edit the [milestone](https://github.com/10up/windows-azure-storage/milestone/9) with release date (in the `Due date (optional)` field) and link to GitHub release (in the `Description` field), then close the milestone.
- [x] Punt incomplete items: If any open issues or PRs which were milestoned for `4.3.4` do not make it into the release, update their milestone to `4.4.0` or `Future Release`.

### PR DESCRIPTION
**Pull request by dkotter**
### Description of the Change

Version bump and release 4.3.4.

Closes #175 

### How to test the Change

Verify changes either locally in VS Code or in the GitHub UI

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,24 @@ All notable changes to this project will be documented in this file, per [the Ke
 
 ## [Unreleased] - TBD
 
+## [4.3.4] - 2022-01-10
+**Note that this release bumps the WordPress minimum version from 4.0 to 5.7 and the PHP minimum version from 5.6 to 7.4.**
+
+### Added
+- Add automated PHPCS scanning (props [@csloisel](https://github.com/csloisel), [@Sidsector9](https://github.com/Sidsector9) via [#169](https://github.com/10up/windows-azure-storage/pull/169)).
```
Comment (jeffpaul) [inline CHANGELOG.md]: This doesn't seem to be what #169 is and I can't seem to find a merged PR that adds PHPCS scanning, but perhaps I'm missing something here if you can help clarify?

Comment (dkotter) [reply CHANGELOG.md]: This should have been #170, fixed that

**Review by jeffpaul — DISMISSED** [review]
looks good, thanks for handling this!

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,24 @@ All notable changes to this project will be documented in this file, per [the Ke
 
 ## [Unreleased] - TBD
 
+## [4.3.4] - 2022-01-10
```
Comment (dkotter) [inline CHANGELOG.md]:
```suggestion
## [4.3.4] - 2023-01-12
```

File: readme.txt [hunk 3]
```diff
@@ -35,6 +35,17 @@ See Settings->Microsoft Azure for more information.
 
 == Changelog ==
 
+= 4.3.4 - 2023-01-10 =
```
Comment (dkotter) [inline readme.txt]:
```suggestion
= 4.3.4 - 2023-01-12 =
```

**Pull request merged**